src/iinit.cpp

```diff
@@ -249,6 +249,59 @@ main( int argc, char **argv ) {
             json_string( my_env.rodsAuthScheme ) );
     }
 
+    // mz_js_20180626+
+    if ( strlen( my_env.rodsClientServerNegotiation ) == 0 ) {
+      if ( !doingEnvFileUpdate ) {
+        doingEnvFileUpdate = true;
+        printUpdateMsg();
+      }
+      printf( "Include default SSL encrypted communication [Y/n]: " );
+      std::string response = "";
+      char inbuf[100];
+      getline( std::cin, response );
+      strncpy( inbuf, response.c_str(), 90 );
+      if ( strncmp( inbuf, "n", 1 ) != 0 || strncmp( inbuf, "N", 1 ) != 0 ) {
```

Comment:
need to think through whether we want this on by default...

or just make it available at first with default still 'N'?

Reply:
I copied those two "magic numbers" 100 and 90 from iquest.cpp line 178, where there is also a Y/n question. 

With respect to the default reply: it's safer to use SSL, but more comfortable to not using it. I prefer to force SSL, since it is my task to provide a secure environment for the scientists especially for medical data.